.github/workflows/main.yml

```diff
@@ -35,9 +37,9 @@ jobs:
           path: |
             .cache
             public
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          key: ${{ runner.os }}-build-${{ hashFiles('**/yarn.lock') }}
```

Comment:
Why did you change this? To bump the cache?

If so, I would suggest to use a "counter" like `-v1-`. Then every time you want to bump the cache, you bump the number.